doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,144 @@
+[[renaming-ha]]
+== Updating Hostnames on Database-Only Appliances
+
+When updating the hostnames of database-only appliances in a cluster, you must also re-configure high availability on the primary and standby database-only appliances.
+
+=== Preparing to Update Appliance Hostnames
+
+Before updating the hostnames of your appliances, complete the following:
+
+. Note the hostnames of the active database (primary) appliance and the standby database appliances in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. On the {product-title_short} (non-database) appliances, stop the failover monitor: 
```

Comment:
is there any reason we want to capture the specification to perform this initial procedure in parentheses? What do you think of just non-database CloudForms appliances?

Reply:
I think this term came about from "CloudForms appliances" (as defined at the beginning of the guide) at times seeming not being totally clear what we were referring to... I think of it like "(non-database)" was a just reminder for the user of what we're talking about there. 
I think this would be a good change to make, but I'll change this throughout the guide so it's more consistent.